src/Voron/Impl/Journal/JournalInfo.cs

```diff
@@ -14,8 +14,13 @@ public unsafe struct JournalInfo
     {
         public const int NumberOfReservedBytes = 3;
 
+        /// <summary>
+        /// Up until version 7.1 - this used to be CurrentJournal,
+        /// this is no longer needed since we are using the existence
+        /// of the actual journal file instead...
+        /// </summary>
         [FieldOffset(0)]
-        public long CurrentJournal;
+        public long Reserved1;
```

Comment:
Will there be an option to _not_ use the shared journals in 7.1? So it would work the same as before 7.1?

Reply:
You can _not_ use it, yes.
However, we are no longer looking at this field any more, even in that scenario